samples/kubernetes/README.md

```diff
@@ -1,16 +1,15 @@
-Monitoring with Prometheus and Kibana using WebLogic Monitoring Exporter
+Monitoring with Prometheus and Kibana using the WebLogic Monitoring Exporter
 =====
 
-Here is the suggested samples of deployments to start Prometheus, Kibana in the kubernetes cluster and the example of the RBAC policy to setup all required permissions to access pods.
+Here are some examples of deployments to start Prometheus and Kibana in the Kubernetes cluster, and an example of the RBAC policy to set up all the required permissions to access pods.
 
-All suggested examples are configured to be deployed in the 'monitoring' namespace, Weblogic Kubernetes Operator is deployed in the 'weblogic-operator' namespace, weblogic domain runs in the 'weblogic-domain' namespace.
+All the examples are configured to be deployed in the `monitoring` namespace. The WebLogic Kubernetes Operator is deployed in the `weblogic-operator` namespace and the WebLogic domain runs in the `weblogic-domain` namespace.
 
-The user can customize the suggested samples to reflect the specific configuration.
+You can customize the examples to reflect a specific configuration.
+
+For more details on how to set up and run Prometheus and Kibana with the WebLogic Monitoring Exporter, see the blog post, [Using Prometheus and Grafana to Monitor WebLogic Server on Kubernetes](https://blogs.oracle.com/weblogicserver/use-prometheus-and-grafana-to-monitor-weblogic-server-on-kubernetes).
 
-For more details how to setup and run Prometheus, Kibana with Weblogic Monitoring Exporter please read [Using Prometheus and Grafana to Monitor WebLogic Server on Kubernetes](https://blogs.oracle.com/weblogicserver/use-prometheus-and-grafana-to-monitor-weblogic-server-on-kubernetes)
 
- 
 ## Copyright
- 
- Copyright (c) 2017-2018 Oracle and/or its affiliates. All rights reserved.
 
+ Copyright (c) 2017-2019 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Copyrights should all be consistent in format
Copyright (c) 2017, 2019, Oracle and/or its affiliates. All rights reserved.

Reply:
Fixed copyright dates.